src/videojs-http-streaming.js

```diff
@@ -226,6 +226,26 @@ const updateVhsLocalStorage = (options) => {
   return objectToStore;
 };
 
+/**
+ * Parses VHS-supported media types from data URIs. See
+ * https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs
+ * for information on data URIs.
+ *
+ * @param {string} dataUri
+ *        The data URI
+ *
+ * @return {string|Object}
+ *         The parsed object/string, or the original string if no supported media type
+ *         was found
+ */
+const expandDataUri = (dataUri) => {
+  if ((/^data:application\/vnd\.videojs\.vhs\+json/i).test(dataUri)) {
```

Comment:
Might be better to use `toLower` on `dataUri` here instead of using regex as that will make it much easier to see what the uri looks like and it is also much more performant. We also might want to require the comma (`,`) at the end. as currently this would match `data:application/vnd.videojs.vhs+json` and then die since there isn't any json data or comma.